giles-eco/src/main/java/edu/asu/diging/gilesecosystem/web/controllers/admin/pages/SystemConfigPage.java

```diff
@@ -14,7 +14,14 @@
     private boolean showGoogleLogin;
     private boolean showMitreidLogin;
     private String gilesFilesTmpDir;
+    private String nepomukAccessToken;
     
+    public String getNepomukAccessToken() {
+        return nepomukAccessToken;
+    }
+    public void setNepomukAccessToken(String nepomukAccessToken) {
+        this.nepomukAccessToken = nepomukAccessToken;
+    }
```

Comment:
Please move them to the end of the class, so that getters/setters are in the same order as the variables as much as possible.